.github/workflows/e2e-ci.yml

```
@@ -98,6 +98,11 @@ jobs:
           FLEET_E2E_NS: fleet-local
         run: |
           ginkgo e2e/single-cluster
+      - name: E2E Tests for keep-resources
+        env:
+          FLEET_E2E_NS: fleet-local
+        run: |
+          ginkgo e2e/keep-resources
```

Should be possible in one line
```suggestion
          ginkgo e2e/single-cluster e2e/keep-resources
```

---

Thanks! done in https://github.com/rancher/fleet/pull/1382/commits/e6275c45de59dab3e5aff760cf54ca6cc1d7e685